ansible/pbTestScripts/updateBoxes.sh

```diff
@@ -41,7 +41,10 @@ if [[ -z "$VBList" ]]; then
 else
   for x in $VBList
   do
-    vagrant box update --box "$x"
+    # Ignore Debian8 for now; See: https://adoptium.slack.com/archives/C53GHCXL4/p1637069847046900
+    if [[ $x != "roboxes/debian8" ]]; then 
```

Comment:
This one definitely should have quotes if the thing on the right of `!=` does :-)
```suggestion
    if [[ "$x" != "roboxes/debian8" ]]; then 
```